types/index.d.ts

```diff
@@ -533,6 +533,7 @@ declare namespace Knex {
     clearOrder(): QueryBuilder<TRecord, TResult>;
     clearHaving(): QueryBuilder<TRecord, TResult>;
     clearCounters(): QueryBuilder<TRecord, TResult>;
+    clear(statement: string): QueryBuilder<TRecord, TResult>;
```

Comment:
Could you narrow down the type of `statement` to the list of supported statements?

Reply:
@kibertoad I don't know typescript, I hope I did it right. I would be grateful for your review.